replica/table.cc

```diff
@@ -2218,7 +2219,6 @@ void table::set_compaction_strategy(sstables::compaction_strategy_type strategy)
     for (auto& updater : cg_sstable_set_updaters) {
         updater.execute();
     }
-    refresh_compound_sstable_set();
```

Comment:
I don't see any changes that do the part which makes compaction finish update the sstable set. Does this code already exists?

Reply:
Yes, since compaction must replace old by new sstables. see compaction_group::update_sstable_sets_on_compaction_completion().